res/controllers/Denon-MC7000-scripts.js

```diff
@@ -562,6 +563,11 @@ MC7000.PadButtons = function(channel, control, value, status, group) {
                 if (engine.getValue("[Sampler" + i + "]", "track_loaded") === 0) {
                     engine.setValue("[Sampler" + i + "]", "LoadSelectedTrack", 1);
                 } else if (engine.getValue("[Sampler" + i + "]", "track_loaded") === 1) {
+                    // STOP PLAYING SAMPLERS ...
+                    for (j = 1; j <=8; j++) {
+                        engine.setValue("[Sampler" + j + "]", "cue_gotoandstop", 1);
+                    }
+                    // ... BEFORE THE ACTUAL SAMPLER TO PLAY GETS STARTED
```

Comment:
NO NEED TO SCREAM :D

Reply:
Hahaha - not my intention - Caps lock activated ;) Cheers!